Gemfile

```diff
@@ -1,7 +1,6 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby ">= 2.6.10"
+ruby ">= 2.7.8"
 
-gem 'cocoapods', '~> 1.12'
-gem 'activesupport', '>= 6.1.7.3', '< 7.1.0'
+gem 'cocoapods', '~> 1.14'
```

Comment:
Looking forward to this change as this version brings support for visionos! 🎉

Reply:
Actually, VisionOS is supported from [1.13.0](https://github.com/CocoaPods/CocoaPods/releases/tag/1.13.0), I think they backported it somehow.

Reply:
@cipolleschi Do you think we can update it to `1.13.0`? I can open a PR for this 👍🏻